windows/klog_main.cpp

```diff
@@ -1,6 +1,7 @@
 #include <Windows.h>
 #include <time.h>
 #include <iostream>
+#include <cstdio>
```

Comment:
`#include <cstudio>` does not work with g++
`#include <stdio.h>`  as suggested in #34 works with g++ and the Visual Studio compiler